guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,7 +4,11 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== postgresql-evr extension no longer required
+
+Installation of the Katello database on remote systems where root access is not available is now possible.
+Only a basic PostgreSQL installation is required.
+With this feature, you can now install on systems like Amazon RDS or Azure Database for PostgreSQL.
```

Comment:
Note to self: since Pulp 3.22 uses the HStore extension (which is why you need `postgresql-contrib`). It's on both  https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-extensions.html and https://learn.microsoft.com/en-us/azure/postgresql/extensions/concepts-extensions-versions#hstore so I'd expect it to indeed work.